kinetic/doc-released-build.yaml

```diff
@@ -0,0 +1,19 @@
+%YAML 1.1
+# ROS buildfarm doc-build file
+---
+documentation_type: released_manifest
+jenkins_job_priority: 89
+jenkins_job_timeout: 30
+notifications:
+  emails:
+  - ros-buildfarm-kinetic@googlegroups.com
+  - esteve+buildfarm@osrfoundation.org
+targets:
+  ubuntu:
+    wily:
+      amd64:
```

Comment:
We should pick the LTS release, xenial, for the doc jobs to ensure support for the whole period. Same for armhf and source builds below.